mod/metrics/set/abstract/04_chart.rb

```diff
@@ -63,7 +63,7 @@ def vega_chart
     output [
       zoom_out_link,
       wrap_with(:div, "",
-                id: id, class: classy("vis"),
+                id: id, class: "#{classy("vis")} _load-vis",
```

Comment:
When we open a modal on a metric page the slot of the main pages receives a "slotReady" for every slot in the modal and loads every time the visualization. With this `_load-vis` class that I remove when it is loaded we save at least at l lot of unnecessary request.